dev/group_vars/micro-integrator.yml

```diff
@@ -75,12 +75,12 @@ performance_tuning_file_list:
 # Set the location the product packages should reside in (eg: "local" in the /files directory, "remote" in a remote location)
 pack_location: local
 #pack_location: remote
-#remote_jdk: "<URL_TO_JDK_FILE>"
-#remote_pack: "<URL_TO_APIM_PACK>"
+#remote_jdk: "<URL_TO_JDK_FILE>" Ex:- "https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz"
```

Comment:
Try to reduce the configuration drift between `dev`, `test`, `staging`, and `production` by using defaults instead of group_vars.